product/models.py

```diff
@@ -0,0 +1,65 @@
+from django.db import models
+
+# Create your models here.
+class Category(models.Model):
+    category = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'categories'
+
+class Product(models.Model):
+    category    = models.ForeignKey('Category', on_delete=models.SET_NULL, null=True)
+    name        = models.CharField(max_length=60)
+    price       = models.CharField(max_length=45)
+    description = models.TextField(max_length=2000)
+    pub_date    = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        db_table = 'products'
+
+class Image(models.Model):
+    product   = models.ForeignKey('Product', on_delete=models.CASCADE)
+    image_url = models.CharField(max_length=1000)
```

Comment:
```suggestion
    image_url = models.URLField(max_length=2000)
```
특별한 기능으로서는 아니지만 장고에 URLField라는것도 있습니다. 그리고 길이는 최소 2000자 이상 주세요.

Reply:
URLField 써야지 생각해놓고 까먹었네요!! 다 수정하겠습니다~